src/style.css

```diff
@@ -78,7 +78,7 @@ a {
   overflow-y: scroll;
 }
 
-.table-row:not(:hover) .hover-only, .table-cell:not(:hover) .cell-hover-only {
+.table-row:not(:hover, :focus-within) .hover-only, .table-cell:not(:hover, :focus-within) .cell-hover-only {
```

Comment:
This particular change allows the Delete button hidden in file browser tables to show up when the user tabs into it. Otherwise the focus ring and tooltip appear around an empty space.

Reply:
Nice!